src/app/game-state/log.service.ts

```diff
@@ -35,7 +36,13 @@ export class LogService {
   followerLog: Log[] = [];
   currentLog: Log[] = [];
 
-  constructor() {
+  constructor(
+    
+    mainLoopService: MainLoopService
+  ) {
+    mainLoopService.frameSubject.subscribe(() => {
+      this.updateLogTopics();
+    });
```

Comment:
Update along with the tick limiter. Tried using {{method()}} but that threw a ton of errors.